apps/fluent-tester/src/TestComponents/FocusZone/FocusZoneTest.tsx

```diff
@@ -86,131 +110,16 @@ const NestedFocusZone: React.FunctionComponent = () => {
-        <Checkboxes />
-      </FocusZone>
-      <FocusZone disabled={true}>
-        <SubheaderText>Disabled FocusZone</SubheaderText>
-        <Checkboxes />
-      </FocusZone>
-    </FocusZoneListWrapper>
-  );
-};
-
-type GridOfButtonsProps = {
-  gridWidth: number;
-  gridHeight: number;
-};
-
-const GridOfButtons: React.FunctionComponent<GridOfButtonsProps> = (props: GridOfButtonsProps) => {
-  return (
-    <View style={focusZoneTestStyles.dashedBorder}>
-      {[...Array(props.gridHeight)].map((_value, heightIndex: number) => {
-        return (
-          <View key={heightIndex} style={focusZoneTestStyles.focusZoneViewStyle}>
-            {[...Array(props.gridWidth)].map((_value, widthIndex: number) => {
-              const gridIndex = heightIndex * props.gridWidth + widthIndex + 1;
-              return (
-                <GridButton key={widthIndex} style={focusZoneTestStyles.focusZoneButton}>
-                  <Text>{gridIndex}</Text>
-                </GridButton>
-              );
-            })}
-          </View>
-        );
-      })}
-    </View>
-  );
-};
-
-interface ISwitchWithLabelProps {
-  label: string;
-  value: boolean;
-  onValueChange: (value: boolean) => void;
-}
-
-function SwitchWithLabel(props: ISwitchWithLabelProps): React.ReactElement {
-  const { label, value, onValueChange } = props;
-  return (
-    <View style={commonTestStyles.switch}>
-      <Text>{label}</Text>
-      <Switch value={value} onValueChange={onValueChange} />
-    </View>
-  );
-}
-
-const FocusZone2D: React.FunctionComponent = () => {
-  const [is2DNav, set2dNav] = React.useState(false);
-  const [isDisabled, setDisabled] = React.useState(false);
-  const [isCircularNav, setIsCircularNav] = React.useState(false);
-  const [direction, setDirection] = React.useState<FocusZoneDirection>('bidirectional');
-
-  return (
-    <View style={commonTestStyles.root}>
-      <View style={commonTestStyles.settings}>
-        <SwitchWithLabel label="2D Navigation" value={is2DNav} onValueChange={set2dNav} />
-        <SwitchWithLabel label="Disabled" value={isDisabled} onValueChange={setDisabled} />
-        <SwitchWithLabel label="Circular Navigation" value={isCircularNav} onValueChange={setIsCircularNav} />
-        <MenuPicker prompt="Direction" selected={direction} onChange={setDirection} collection={directionCollection} />
-      </View>
-
-      <FocusZoneListWrapper>
-        <FocusZone disabled={isDisabled} use2DNavigation={is2DNav} focusZoneDirection={direction} isCircularNavigation={isCircularNav}>
-          <GridOfButtons gridWidth={3} gridHeight={3} />
-        </FocusZone>
-      </FocusZoneListWrapper>
-    </View>
-  );
-};
-
 const focusZoneSections: TestSection[] = [
   {
     name: 'Common FocusZone Usage',
     component: CommonUsageFocusZone,
+    testID: FOCUSZONE_TESTPAGE,
   },
   {
     name: 'Directional FocusZone Usage',
-    testID: FOCUSZONE_TESTPAGE,
     component: DirectionalFocusZone,
   },
-  {
```

Comment:
Why did we remove this section? Typically we want our partner example scenarios to be separate from our E2E testing examples.